x-pack/plugins/graph/public/components/workspace_layout/workspace_top_nav_menu.tsx

```diff
@@ -162,6 +162,9 @@ export const WorkspaceTopNavMenu = (props: WorkspaceTopNavMenuProps) => {
           ownFocus: true,
           className: 'gphSettingsFlyout',
           maxWidth: 520,
+          'aria-label': i18n.translate('xpack.graph.settings.ariaLabel', {
+            defaultMessage: 'Settings flyout',
```

Comment:
Not sure if "flyout" is commonly used. Maybe just "Settings" or we could add `aria-labelledby="<id of the title element>"`.

Reply:
thx Julia, makes sense, will improve this 👍

Reply:
Relabeled to "Settings"